try:
    result = 10/0
except:
    print("Execute me if try has some error")